doc/conf.py

```diff
@@ -42,6 +42,11 @@
     print "ipython: %s, %s" % (IPython.__version__, IPython.__file__)
 except ImportError:
     print "no ipython"
+try:
+    import seaborn
```

Comment:
Happens when I run `make clean` and then `make html`. Ran into something like this earlier and fixed it by using `matplotlib.use('Agg')`. PR coming along shortly.